apps/odyssey-stats/src/app.tsx

```diff
@@ -44,6 +44,12 @@ async function AppBoot() {
 			...initialState.currentUser,
 			user: { ...initialState.currentUser.user, localeSlug },
 		},
+		sites: {
+			...initialState.sites,
+			features: {
+				...initialState.sites.features,
```

Comment:
Does it change anything here? Seems not 🤔

Reply:
Indeed, we just need sites that include features.